packages/framework-core/src/booster-token-verifier.ts

```diff
@@ -67,14 +67,26 @@ export class BoosterTokenVerifier {
     const rolesClaim = this.config.tokenVerifier?.rolesClaim || 'custom:role'
     const role = decodedToken[rolesClaim]
     const roleValue = Array.isArray(role) ? role[0] : role
+    const customParams = this.getCustomParameters(decodedToken)
 
     return {
       id,
       username,
       role: roleValue?.trim() ?? '',
+      ...customParams,
     }
   }
 
+  private getCustomParameters(decodedToken: Record<string, unknown>): Record<string, unknown> {
+    return Object.keys(decodedToken)
+      .filter((key: string) => key.includes('custom:') && key !== 'custom:role')
+      .reduce((obj: Record<string, unknown>, key: string) => {
+        const cleanKey = key.split(':')[1]
+        obj[cleanKey] = decodedToken[key]
+        return obj
+      }, {})
+  }
```

Comment:
Hmm I think that carries only role info and nothing else. It's created here I think: https://github.com/boostercloud/rocket-auth-aws-infrastructure/blob/main/src/lambdas/sign-up.ts#L60

At this point, the token has been already verified (that's why I'm using the `decodedToken` object) but let me know if I'm missing something 🙏

Reply:
It carries more info, including any custom claim that the auth provider the user is using might add. This is the reason I think it could be more useful (and easier on our side) to include all.
Take into account that the reason to include only those tockens prefixed with `custom:` is arbitrary. It is true that Cognito uses those, but other auth providers or proxies use others. For example, Hasura uses something like `x-hasura-blahblah`. If we just include those with the `custom:` prefix, we would be limiting a lot the usefulness.

Reply:
BTW, I didn't fully understand your last sentence